copter/source/docs/traditional-helicopter-tuning.rst

```diff
@@ -315,86 +260,28 @@ discrepancy between desired and actual pitch and roll attitude when the
-the roll angle that you had to hold with the stick to keep the helicopter from
-drifting. Enter this value in the :ref:`ATC_HOVR_ROL_TRM<ATC_HOVR_ROL_TRM>` parameter in centidegrees.
-For a CW turning main rotor if it took 3.5 degrees of right roll to compensate,
-enter 350. Negative values are for a CCW turning main rotor that requires left
-roll to compensate.
-
-**Important Note** - do not use the radio trims at all. Make sure they are
-centered. 
-
-After setting the :ref:`ATC_HOVR_ROL_TRM<ATC_HOVR_ROL_TRM>` now hover the helicopter again. If it still
-drifts make small adjustments to the :ref:`SERVO1_TRIM<SERVO1_TRIM>` , :ref:`SERVO2_TRIM<SERVO2_TRIM>` and :ref:`SERVO3_TRIM<SERVO3_TRIM>` .
-The chances of getting the swashplate perfectly level during bench setup is very
-low and this dynamic tuning is needed to trim the helicopter. If it requires
-large deviation from your original ``SERVOx_TRIM`` values it is likely you have a CG
-problem, or your initial setup when leveling the swashplate was not very
-accurate.
-
-Your helicopter is now trimmed properly. This trimming procedure makes the
-difference between a helicopter that is difficult to handle vs one that flies
-with true scale quality and handling. 
-
-Adjusting I-gains For High-Speed Autonomous Flight
-==================================================
-Prepare a mission with your ground station software that will fly the 
-helicopter, preferably in a figure-8 pattern to make both right and left turns,
-at a speed of 6 m/s. Fly the helicopter on this mission, pull the logs from the
-microSD card and look at the AHRS desired vs actual pitch, roll and yaw
-attitudes in dynamic flight. They should track within 1-2 degrees. If they do
-not, increase the ``ATC_RAT_xxx_I`` value for that axis until they do.
-
-Now, fly the same mission, but at higher speed of 9-10 m/s, and analyze the logs
-the same way. Make further adjustments to the I-gains and IMAX values as
-required. It is not clear what I-gain values will be required as no two
-helicopters are the same. But I-gain values from 0.25 - 0.38 are common in pitch
-and roll, and 0.18 - 0.30 in yaw. IMAX values of 0.40 - 0.45 are common, however
-refer to the 'Setting the I gain, IMAX, and ILMI' section on how to determine
-what the IMAX value should be.
+Tuning a Flybar Equipped Head
+=============================
+
+With a flybar head, where the linkage rate is normally lower, it is recommended
+to start with 0.22 FF for both pitch and roll and you will likely have to go
+higher with FF. But for a flybarless head, FF shouldn't be more than 0.22 
+unless you have really really slow servos or slow linkage rate. With all 
+helicopters, the FF gain compensates for differences in servo and linkage
+speed. 
+
+**Note on Tuning Flybar Helicopers** - for flybar heads the flybar is a
+mechanical version of the rate PID loop. So flybar is tuned using only FF in
+pitch and roll. The rate D and P gains mentioned in the next topic below are
```

Comment:
It references rate P and D gain sections which are not after this section.  I would just remove the reference and just tell the reader to set the Rate P and D gains to zero.

Reply:
fixed.